incipyt/__main__.py

```diff
@@ -26,12 +30,39 @@
 @click.option("-v", "--verbose", count=True)
 @click.option("-s", "--silent", count=True)
 @click.version_option()
+@click.option(
+    "--vcs",
+    required=True,
+    show_default=True,
+    default="Git",
+    type=click.Choice(["None", "Git"], case_sensitive=False),
```

Comment:
```suggestion
    type=click.Choice(["", "Git"], case_sensitive=False),
```

Reply:
just an idea

Reply:
If we do that, then the command line becomes:
`incipyt --vcs "" folder` or `incipyt --vcs '' folder`
Instead of
`incipyt --vcs none folder`
Do you find the first one really better?

Reply:
I do, that's the proper way to void an option, at least it is what my experience is telling me. Btw `--vcs= ` also works